test/lib/ansible_test/_internal/config.py

```diff
@@ -298,6 +298,7 @@ def __init__(self, args):
 
         self.collect_only = args.collect_only  # type: bool
         self.num_workers = args.num_workers  # type: int
+        self.pdb = args.pdb == 'yes'
```

Comment:
What is the purpose of this new config? It isn't used anywhere.

Reply:
Thank you for catching my silly mistake. I have passed this in.
Tested like this  `strace -f -c -s 9999 python3  bin/ansible-test units --pdb` and `strace -f -c -s 9999 python3  bin/ansible-test units ` and confirmed it was passed through.